full_build.sh

```diff
@@ -28,13 +28,33 @@ git checkout composer.json
 rm -rf rector-build/packages-tests rector-build/rules-tests rector-build/tests rector-build/bin/generate-changelog.php rector-build/bin/validate-phpstan-version.php
 
 sh build/downgrade-rector.sh rector-build
-sh build/build-rector-scoped.sh rector-build rector-prefixed-downgraded
 
-cd rector-prefixed-downgraded
+cd rector-build
+
+# avoid syntax error in php 7.1 and 7.2
+rm rector.php
+
 cp ../build/target-repository/bootstrap.php .
 cp ../preload.php .
-bin/rector list --ansi
-bin/rector process vendor/symfony/string/Slugger/ --dry-run
+
+# Update to your local php 7.1 path
+/opt/homebrew/Cellar/php@7.1/7.1.33_4/bin/php bin/rector list --ansi
 
 cd ..
-rm -rf rector-prefixed-downgraded
\ No newline at end of file
+rm -rf rector-build
+
+# Prefixed build only works on PHP < 8.1 now, so only able to run on CI.
```

Comment:
ref : 

https://github.com/rectorphp/rector-src/blob/650a8d6a13ee2edf5d9aa2b3aa0cfada95589fe4/.github/workflows/build_scoped_rector.yaml#L68

for it, we currently run prefixing from php 8.0 on CI after code downgraded as php-scoper .phar still doesn't support php 8.1